CMakeLists.txt

```diff
@@ -40,7 +57,10 @@ if(CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
               ${CMAKE_CURRENT_SOURCE_DIR}/cmake/FindFilesystem.cmake
         DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/tmp)
 
-    install(TARGETS ${PROJECT_NAME} EXPORT tmp-targets)
+    install(
+      TARGETS ${PROJECT_NAME}
+      EXPORT tmp-targets
+      FILE_SET CXX_MODULES DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
```

Comment:
The only example I could find installs `.cppm` files to `share/`:
```shell
$ find . -name '*.cppm'         
./llvm/18.1.5/share/libc++/v1/std.compat.cppm
./llvm/18.1.5/share/libc++/v1/std.cppm
```

No idea how to use them tho